### DISCUSSION TITLE
my account is flagged

### DISCUSSION THREAD

**Discussion by Shaik-Mastan-Sharief** [community/community#35707] [New to GitHub] [answered]
hi to everyone the main purpose of this discussion is my account is flagged so help me

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as the Community is unable to assist in situations like this :)

**Reply by GarfQC** [reply]
that's exactly what I'm doing and it gets me back here
this is redundant

**Reply by yalla-shoot-smart** [reply]
Hello my friend
Please look at my account also it has been flagged I don't know why
I opened a ticket and no response yet, her number is #1866989

**Comment by SSSAwais** [comment]
Hello buddy I hope you are doing good but i m not as my account suddenly got flagged. Yesterday it was all OK but today it is flagged please do the need full and make it unflagged ASAP. I really appreciate you for this act. Thanks

**Comment by sssghulamrasool** [comment]
![image](https://user-images.githubusercontent.com/115653920/196611335-80f20f5d-b7d0-4856-a38b-f62f9cd7d927.png)

Hello buddy I hope you are doing good but I m not as my account suddenly got flagged. Yesterday it was all OK but today it is flagged please do the need full and make it unflagged ASAP. I really appreciate you for this act. Thanks

**Reply by SARAN-SMART-2004** [reply]
Same Issue What Can i do

**Comment by nurfaden** [comment]
Hi, my account has been flagged suddenly. I'm a new user, I don't understand why.

**Comment by gaparicioAlphas** [comment]
Hello, recently I create at github account because my new job request it, but they can't share the repository because that told me the account has been flagged, why happened it? Thanks alot.

![imagen](https://user-images.githubusercontent.com/116763773/198144467-98189001-4743-490a-9d95-17db0602e2de.png)

**Comment by Mai-Thanh-Thuan-15-4-2** [comment]
> hi to everyone the main purpose of this discussion is my account is flagged so help me

**Comment by antonyponnavin** [comment]
Sorry about that!

Our spam-hunting bots usually do a good job, but from time to time they get things wrong. Today was one of those times, and I’m sorry you were on the receiving end of their mistake. While we aren't let into too many of the workings of their silicon brains, we are able to teach them the error of their ways

Please remove flag to my account..

**Comment by FootballINU** [comment]
Hello, recently I create at github account because my new job request it, but they can't share the repository because that told me the account has been flagged, why happened it? Thanks alot.

**Comment by Taras9441** [comment]
hi to everyone my account is flagged so help me

**Comment by tommyleong123** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as the Community is unable to assist in situations like this :)

**Comment by SandraMadarnas** [comment]
Hola! Hace poco cree una cuenta en Gitpod y Github. Es para un curso que estoy realizando. Al poco de registrarme me sale un mensaje que dice "Your account has been flagged." No entiendo porque y me urge arreglar este problema para poder continuar con mi curso.

Muchas Gracias de antemano.

**Comment by lockedech** [comment]
Hello buddy I hope you are doing good but I m not as my account suddenly got flagged. Yesterday it was all OK but today it is flagged please do the need full and make it unflagged ASAP. I really appreciate you for this act. Thanks

**Comment by OleksandrVasilenko** [comment]
"Ваш аккаунт был помечен.
Из-за этого ваш профиль скрыт от общественности. Если вы считаете, что это ошибка, [обратитесь в службу поддержки] , чтобы проверить статус вашей учетной записи."
Добрый день помогите пожалуйста, создал вместо старого акаунта новый и акаунт помечен.

**Comment by yalla-shoot-smart** [comment]
Hello, please look again in the account, as this is specific to our site, Yalla Shoot Smart, and we create special tools for the site through you.

**Comment by mikelakoneli** [comment]
Hello buddy I hope you are doing good but I m not as my account suddenly got flagged. Yesterday it was all OK but today it is flagged please do the need full and make it unflagged ASAP. I really appreciate you for this act. Thanks

**Comment by roxchatdev** [comment]
What does it mean at all?
shall I do something with this "flag"?)
anybody...

**Comment by xavierfontanilles** [comment]
hello there

I still have this " Your account has been flagged."
Again, I can not follow my training payed 1400$ because I have not access to https://kitt.lewagon.com/camps/1082
Can you please fix the problem ?

Thank you 
Xavier

**Comment by thetungle** [comment]
Hello buddy I hope you are doing good but I m not as my account suddenly got flagged.

**Comment by irovnyhka** [comment]
![image](https://user-images.githubusercontent.com/118622510/203223830-b554ceb7-18cb-47ed-8ed7-bd0fe734d8dd.png)

**Comment by 2captcha** [comment]
Hi!
My account was flagged about a week ago, no response in support ticket during 6 days. How to resolve this issue?

**Comment by heatherleeann** [comment]
My account is flagged

**Reply by nurfaden** [reply]
Mine is still flagged too! Unfortunately, it's been a month, I'm still waiting for the response.

**Comment by queenofcorgis** [comment]
Looks like quite a few folks have reached out about flagged accounts. As a reminder, the community is unable to assist in account-specific matters. The best (and only) place to go for flagged account assistance is our [Support page](https://support.github.com/contact) where you'll open a ticket. I'm going to lock this post as I mentioned the community is unable to intervene further.
